package.json

```diff
@@ -46,5 +48,8 @@
   },
   "nx": {
     "includedScripts": []
+  },
+  "dependencies": {
+    "@jupyterlab/mathjax-extension": "^4.0.12"
```

Comment:
Should it go to `packages/jupyter-ai/package.json` rather than top-level `package.json`? This is where all other dependencies are defined.

Reply:
That's a point, thank you. I removed dependency on `mathjax-extension` completely and use jupyterlab's`rmRegistry.latexTypesetter` if it exists instead. It seems like a better architecture when our goal is to match notebook cell rendering.